packages/strapi-hook-bookshelf/lib/buildQuery.js

```diff
@@ -53,6 +31,165 @@ const buildQuery = ({ model, filters }) => qb => {
   }
 };
 
+/**
+ * Add joins and where filters
+ * @param {Object} qb - knex query builder
+ * @param {Object} model - Bookshelf model
+ * @param {Array<Object>} whereClauses - an array of where clause
+ */
+const buildJoinsAndFilter = (qb, model, whereClauses) => {
+  const aliasMap = {};
+  /**
+   * Returns an alias for a name (simple incremental alias name)
+   * @param {string} name - name to alias
+   */
+  const generateAlias = name => {
+    if (!aliasMap[name]) {
+      aliasMap[name] = 1;
+    }
+
+    return `${name}_${aliasMap[name]++}`;
```

Comment:
Just asking... but is it a good practice to increment a number inside a templated string?